spec/system/admin/orders_spec.rb

```diff
@@ -384,6 +384,26 @@
           login_as_admin_and_visit spree.admin_orders_path
         end
 
+        it "can bulk send invoice for 2 orders" do
+          page.find("#listing_orders tbody tr:nth-child(1) input[name='order_ids[]']").click
+          page.find("#listing_orders tbody tr:nth-child(2) input[name='order_ids[]']").click
+
+          page.find("span.icon-reorder", text: "ACTIONS").click
+          within ".ofn-drop-down-with-prepend .menu" do
+            page.find("span", text: "Send Invoices").click
+          end
+
+          expect(page).to have_content "Are you sure you want to proceed?"
```

Comment:
I think it's good that this doesn't test the entire string, because when picky people (like me) make minor changes, it doesn't have to be updated ;)

But I noticed that this is quite generic and could pass if a different translated string was used. So maybe it is necessary after all.

```suggestion
          expect(page).to have_content "This will email invoices to all selected customers."
          expect(page).to have_content "Are you sure you want to proceed?"
```

Reply:
Assertion added!